### DISCUSSION TITLE
Markdown tab length?

### DISCUSSION THREAD

**Discussion by MWP** [sebthom/previewer-eclipse-plugin#4] [Q&A] [answered]
Is there as way to set markdown tab length?
It appears that tab=8 chars, i need it to be set at 4.
Thanks in advance.

**Comment by sebthom** [comment, marked as answer]
Please give the latest release a try. It now uses the tab width defined under `Preferences > General > Editors > Text Editors > Displayed tab width`

**Reply by MWP** [reply]
Perfect, thank you very much :)
